<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SDBuildOptions</class>
 <widget class="QWidget" name="SDBuildOptions">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimDSP - Build Options</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../simdspresources.qrc">
    <normaloff>:/resources/images/icons/simdsp.png</normaloff>:/resources/images/icons/simdsp.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>9</number>
   </property>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayoutInclude" columnstretch="3,1" columnminimumwidth="0,0">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelInclude">
       <property name="text">
        <string>Include Path</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="addInclude">
         <property name="toolTip">
          <string>Add Include Path</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../simdspresources.qrc">
           <normaloff>:/resources/images/icons/add.png</normaloff>:/resources/images/icons/add.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editInclude">
         <property name="toolTip">
          <string>Edit Include Path</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../simdspresources.qrc">
           <normaloff>:/resources/images/icons/edit.png</normaloff>:/resources/images/icons/edit.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeInclude">
         <property name="toolTip">
          <string>Remove Include Path</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../simdspresources.qrc">
           <normaloff>:/resources/images/icons/remove.png</normaloff>:/resources/images/icons/remove.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QListWidget" name="listWidgetInclude">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Build Options</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayoutLibraries" columnstretch="3,1" columnminimumwidth="0,0">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="addLibrary">
         <property name="toolTip">
          <string>Add Library Path</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../simdspresources.qrc">
           <normaloff>:/resources/images/icons/add.png</normaloff>:/resources/images/icons/add.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editLibrary">
         <property name="toolTip">
          <string>Edit Library Path</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../simdspresources.qrc">
           <normaloff>:/resources/images/icons/edit.png</normaloff>:/resources/images/icons/edit.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeLibrary">
         <property name="toolTip">
          <string>Remove Library Path</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../simdspresources.qrc">
           <normaloff>:/resources/images/icons/remove.png</normaloff>:/resources/images/icons/remove.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelLibraries">
       <property name="text">
        <string>Libraries Path</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QListWidget" name="listWidgetLibraries">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QGridLayout" name="gridLayoutLib" columnstretch="3,1">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelLib">
       <property name="text">
        <string>Link Libraries</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="addLib">
         <property name="toolTip">
          <string>Add Link Library</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../simdspresources.qrc">
           <normaloff>:/resources/images/icons/add.png</normaloff>:/resources/images/icons/add.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editLib">
         <property name="toolTip">
          <string>Edit Link Library</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../simdspresources.qrc">
           <normaloff>:/resources/images/icons/edit.png</normaloff>:/resources/images/icons/edit.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeLib">
         <property name="toolTip">
          <string>Remove Link Library</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../simdspresources.qrc">
           <normaloff>:/resources/images/icons/remove.png</normaloff>:/resources/images/icons/remove.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QListWidget" name="listWidgetLib">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelStandard">
       <property name="text">
        <string>C++ language standard</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxStd">
       <item>
        <property name="text">
         <string>C++11 ISO C++ language standard [c++11]</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>C++14 ISO C++ language standard [c++14]</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>C++0x ISO C++ language standard [c++0x]</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1998 ISO C++ language standard [c++98]</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../simdspresources.qrc"/>
 </resources>
 <connections/>
</ui>
